jep/223/README.adoc

```diff
@@ -86,8 +86,44 @@ The changes to the existing permissions are detailed in the diagram below.
 
 image::Targetted_permission_model.png[Design of the new permission model.]
 
+Specifically, for an installation of Jenkins v2.207 with the default suggested plugins, the following configuration items would be available to a user with the `Overall/Manage` permission:
+
+* `<JENKINS_URL>/manage`
+** `Configure System`
+** `Reload Configuration From Disk`
+** `System Information`
+** `Load Statistics`
+** `Jenkins CLI`
+** `About Jenkins`
+** `Manage Old Data`
+** `Prepare For Shutdown`
+* `<JENKINS_URL>/configure`
+** `Home Directory` (_read only_)
+** `System Message`
+** `Quiet Period`
+** `SCM checkout retry count`
+** `Restrict project naming`
+** `Pipeline Default Speed/Durability Level`
+** `Enable Anonymous Usage Statistics`
+** `Enable BuildStep Action`
```

Comment:
Pretty sure this is contributed by a plugin, specifically build-timeout, and makes little sense to readers without stating it explicitly.